.github/workflows/scp-items.yml

```diff
@@ -0,0 +1,92 @@
+name: Crawl SCP Wiki
+
+on:
+#   workflow_dispatch:
+#   schedule:
+#     - cron: "0 0 * * *"
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/scp-items.yml
+
+permissions:
+  contents: write
+
+jobs:
+  update-main-scp:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Clone this Repository"
+        uses: actions/checkout@v6
+        with:
+          path: scp-api
```

Comment:
scp-api is a different repo than this one. It looks like this was copy/pasted from the other repo here.

I love the idea of having this test in here, but lets make sure we're pulling the right repos.